tests/end2end/playwright/draw.spec.js

```diff
@@ -301,6 +302,57 @@ test.describe('Draw', () => {
         await expect(page.locator('.ol-tooltip.ol-tooltip-static')).toHaveText('3.3 km34.27 km2');
     })
 
+    test('Measure - length and angle constraints', async ({ page }) => {
```

Comment:
New tests should also be tagged `@readonly` : https://github.com/3liz/lizmap-web-client/tree/master/tests#writing-tests